internal/metrics/prometheus.go

```diff
@@ -129,20 +161,43 @@ func NewCtrlFuncMetrics() *CtrlFuncMetrics {
 		[]string{SuccessKey, ProtocolKey, DataplaneKey},
 	)
 
+	controllerMetrics.ConfigPushSuccessTime = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: MetricNameConfigPushSuccessTime,
+			Help: fmt.Sprintf("The time of the last successful configuration push. "+
+				"`%s` describes the dataplane that was the target of the configuration push.",
+				DataplaneKey,
+			),
+		},
+		[]string{DataplaneKey},
+	)
+
 	if ok := metrics.Registry.Unregister(controllerMetrics.ConfigPushCount); ok {
 		fmt.Printf("Unregistered %s\n", MetricNameConfigPushCount)
 	}
+	if ok := metrics.Registry.Unregister(controllerMetrics.ConfigPushBrokenResources); ok {
+		fmt.Printf("Unregistered %s\n", MetricNameConfigPushCount)
```

Comment:
I'm not sure why I left the `fmt.Printf()` calls here. Those are not necessary and can clobber the logs so we might as well get rid of them and ignore the return value of `.Unregister()`